js/views/account.js

```diff
@@ -53,6 +65,45 @@ define(function(require) {
 		toggleCollapse: function() {
 			this.collapsed = !this.collapsed;
 			this.render();
+			this.listenTo(Radio.ui, 'document:click', function(event) {
```

Comment:
@Gomez this code should catch that. @tahaalibra can you have a look at this please?